packages/react-native/src/notifier.js

```diff
@@ -97,32 +98,45 @@ const Bugsnag = {
       Bugsnag._client._logger.warn('Bugsnag.start() was called more than once. Ignoring.')
       return Bugsnag._client
     }
+
     if (!isDebuggingRemotely) {
       Bugsnag._client = createClient(opts)
       return Bugsnag._client
-    } else {
-      console.warn(REMOTE_DEBUGGING_WARNING)
-      let initialised = false
-      const stubClient = {}
-      CLIENT_METHODS.reduce((accum, m) => {
-        stubClient[m] = new Proxy(() => {
-          console.warn(
-            `This call to Bugsnag.${m}() is a no-op because remote debugging is enabled and Bugsnag has not yet initialized`
-          )
-        }, {
-          apply: (target, thisArg, argumentsList) => {
-            if (!initialised) return Reflect.apply(target, thisArg, argumentsList)
-            return Reflect.apply(Bugsnag._client[m], Bugsnag._client, argumentsList)
+    }
+
+    console.warn(REMOTE_DEBUGGING_WARNING)
+
+    let initialised = false
+
+    const stubSchema = { ...schema }
+    delete stubSchema.apiKey
+    const stubClient = new Client({
+      ...opts,
+      autoTrackSessions: false,
+      autoDetectErrors: false,
+      enabledBreadcrumbTypes: []
+    }, stubSchema, internalPlugins, { name, version, url })
+
+    CLIENT_METHODS.map((m) => {
+      if (/^_/.test(m)) return
+      stubClient[m] = new Proxy(stubClient[m], {
+        apply: (target, thisArg, args) => {
+          if (!initialised) {
+            console.log(`Synchronous call to Bugsnag.${m}()`)
```

Comment:
I'm not sure I'd understand what this message is trying to say if I didn't know the context; how about something closer to the old message?

```suggestion
            console.log(`Synchronous call to Bugsnag.${m}() may not work because remote debugging is enabled and Bugsnag has not yet initialized`)
```

Reply:
I think it's pretty clear in context, because you get it right after the big block of warning text:

![image](https://user-images.githubusercontent.com/609579/105155704-d239d600-5b02-11eb-9a6d-117f011d22c3.png)

Reply:
Ah yep, that looks good